scripts/ingests/ingest_BYW_2024.py

```diff
@@ -0,0 +1,203 @@
+from scripts.ingests.ingest_utils import *
+from scripts.ingests.utils import *
+from astropy.table import Table
+import astropy.units as u
+from astropy.coordinates import Angle
+
+SAVE_DB = True  # save the data files in addition to modifying the .db file
+RECREATE_DB = True  # recreates the .db file from the data files
+# LOAD THE DATABASE
+db = load_simpledb('SIMPLE.db', recreatedb=RECREATE_DB)
+
+
+# live google sheet
+link = "https://docs.google.com/spreadsheets/d/1JFa8F4Ngzp3qAW8NOBurkz4bMKo9zXYeF6N1vMtqDZs/edit#gid=0"
+
+columns = ['source', 'ra', 'dec', 'epoch', 'equinox', 'shortname', 'reference', 'other_ref', 'comments']
+byw_table = Table.read(link, format='ascii', data_start=2, data_end=90, header_start=1, names=columns, guess=False,
+                           fast_reader=False, delimiter=',')
+
+data_columns = ['CWISE J000021.45-481314.9', '0.0893808', '-48.2208077', '2015.4041', 'ICRS', 'CWISE 0000-4813', 'Rothermich', 'None', 'None']  # columns with wanted data values
+
+print(byw_table)
+
+# replacing empty values ('cdots') with None
+for column in data_columns:
+    byw_table[column][np.where(byw_table[column] == 'cdots')] = None
+
+def ingest_source(db, source, reference=None, ra=None, dec=None, epoch=None, equinox=None, 
```

Comment:
This function is already defined in ingest_utils and imported. You just have to **use** the function. (Same as you used ingest_proper_motions.) Delete lines 28 onward.